.travis.yml

```diff
@@ -13,7 +13,8 @@ install:
 
 script:
   - pytest --cov=./libp2p tests/
-  - pylint --rcfile=.pylintrc libp2p tests
+  - black --check examples libp2p tests setup.py
```

Comment:
Since we have mypy now, could you rearrange the order to the following?
```
    black --check examples libp2p tests setup.py
    flake8 libp2p/ tests/
    mypy ...
```